contrib/azurerm/apply-rg.sh

```diff
@@ -9,10 +9,10 @@ if [ "$AZURE_RESOURCE_GROUP" == "" ]; then
     exit 1
 fi
 
-if [ $(az &>/dev/null) ] ; then
+if [[ $(az &>/dev/null) ]] ; then
```

Comment:
Why are there brackets here? This line should probably be just
`if az &>/dev/null; then`

It's a dirty hack to treat the output of a command as a parsable string without an operand. If you want to test if the command has a nonempty result, the line should read:
`if [ -n "$(az &>/dev/null)" ]`
or
`if [[ -n "$(az &>/dev/null)" ]]` 
(single or double braces are okay)

But my hunch is that the first option is the right one.